spec/deps/git.go

```diff
@@ -100,14 +100,15 @@ func (gs *Git) Remote() string {
 
 // regular expressions for matching package uris
 const (
-	gitSSHExp     = `ssh://git@(?P<host>.+)/(?P<user>.+)/(?P<repo>.+).git`
-	gitSCPExp     = `^git@(?P<host>.+):(?P<user>.+)/(?P<repo>.+).git`
-	githubSlugExp = `github.com/(?P<user>[-_a-zA-Z0-9]+)/(?P<repo>[-_a-zA-Z0-9]+)`
+	gitSSHExp = `ssh://git@(?P<host>.+)/(?P<user>.+)/(?P<repo>.+).git`
+	gitSCPExp = `^git@(?P<host>.+):(?P<user>.+)/(?P<repo>.+).git`
+	// The long ugly pattern for ${host} here is a generic pattern for "valid URL with zero or more subdomains and a valid TLD"
+	gitSlugExp = `(?P<scheme>https://)?(?P<host>[a-zA-Z0-9][a-zA-Z0-9-\.]{1,61}[a-zA-Z0-9]\.[a-zA-Z]{2,})/(?P<user>[-_a-zA-Z0-9]+)/(?P<repo>[-_a-zA-Z0-9]+)`
 )
 
 var (
-	gitSSHRegex     = regexp.MustCompile(gitSSHExp)
-	githubSlugRegex = regexp.MustCompile(githubSlugExp)
+	gitSSHRegex  = regexp.MustCompile(gitSSHExp)
```

Comment:
Are these used at all? Feel like I forgot to remove them in my last PR

Reply:
I might have goofed up the merge of master, the conflicts threw me for more of a loop than usual.  I'll check it out :-)

Reply:
Looks like the SSH one was still in master and I missed it, which led me to leave mine there too. :-) Removed both!